.github/CONTRIBUTING.md

```diff
@@ -73,13 +73,23 @@ In case you are suggesting a new feature, we will match your idea with our curre
     yarn install
     ```
 
--   Bootstrap all the submodules before building for the first time
+-   Bootstrap all the submodules before building for the first time.
 
     ```bash
     yarn bootstrap
     yarn build
     ```
 
+-   Create a symlink of package. [Learn More](https://classic.yarnpkg.com/en/docs/cli/link/)
+
+    ```bash
+    yarn link
+
+    //now go to the location you want to test the package.
+
+    yarn link webpack-cli
+    ```
```

Comment:
I don't do it and all works fine, what is a problem?

Reply:
Actually some new contributors get confused with how to test their changes. That's why added this step. 
[For example Refer this comment](https://github.com/webpack/webpack-cli/issues/1165#issuecomment-599445409)

Reply:
Good to have so anyone can look in docs instead of asking someone.

Reply:
I think we're good as is.